frontend/public/models/yaml-templates.ts

```diff
@@ -808,6 +808,15 @@ spec:
     apiVersion: machine.openshift.io/v1beta1
     kind: MachineSet
     name: worker
+`).setIn([referenceForModel(k8sModels.ConsoleLinkModel), 'default'], `
+apiVersion: console.openshift.io/v1
+kind: ConsoleLink
+metadata:
+  name: example
+spec:
+  href: 'https://www.example.com'
+  location: HelpMenu
+  text: Additional help menu link
```

Comment:
nit: We probably want title case in the YAML template since the other menu items are title case